frontend/src/app/index.css

```
@@ -14,6 +14,14 @@
 }
 
 :root {
+  --font-sans: 'Pretendard Variable', -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif;
```

Stylelint (17.7.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)